docs/data/material/components/accordion/ControlledAccordions.tsx

```diff
@@ -21,10 +44,8 @@ export default function ControlledAccordions() {
           aria-controls="panel1bh-content"
           id="panel1bh-header"
         >
-          <Typography sx={{ width: '33%', flexShrink: 0 }}>
-            General settings
-          </Typography>
-          <Typography sx={{ color: 'text.secondary' }}>I am an accordion</Typography>
+          <AccordionSummaryText>General settings</AccordionSummaryText>
```

Comment:
Reading `AccordionSummaryText` makes me think this component is part of the `Accordion` API. Can we find another name that doesn't use the "Accordion" prefix?

Reply:
Make sense. Changed it to `SummaryText`.